File: Easy/1_TwoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    Loop1:
    for (i=0; i<nums.length; i++){
        Loop2:        
        for (j =0; j< nums.length; j++){

            if ( (i!=j) &&(nums[i]+ nums[j] == target)){
                
                return ([i, j])
            }
        }
    }
};

File: Easy/1_TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        s = {}
        for i in range(len(nums)):
            if((target-nums[i])in s):
                return[s.get(target- nums[i]), i]
            s[nums[i]] = i